README.md

```diff
@@ -262,10 +257,13 @@ that you wish the world to use:
 
 ### Adjusting global server properties
 Default values in the script can be overridden by adding certain properties to one
-of the `mscs.defaults` files. The `mscs.defaults` files can be found found in one
+of the `mscs.defaults` files. The `mscs.defaults` files needs to be created in one
```

Comment:
I wonder if we should have the installer create `/opt/mscs/mscs.defaults` and have the options commented out. That or include a file the user can copy and rename.